docs/getting-started/migrate/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Migrate from the legacy Shopify App CLI
+section: getting-started
+redirect_from: "/migrate/"
+---
+
+As of version 0.9.0, Shopify CLI is installed and managed as a software package, instead of as a git repository. If you're using an older version, you’ll need to perform a one-time migration to keep using the CLI. Follow these steps to remove the old version and reinstall as a package.
```

Comment:
There's a mix of ASCII and “proper” punctuation in this file. Let’s “educate” all of the quotation marks and apostrophes?